### PR TITLE
Penpal agent

### PR DESCRIPTION
**Pull request by amritap-ef**
Fixes some formatting issues and agent now correctly updates when customisable options changes

### CODE REVIEW

File: generative_agents/demo/penpal_app.py [hunk 1]
```diff
@@ -1,105 +0,0 @@
...
-
-log = logger.init("Streamlit app")
-
-
-def generate_response(user_input):
-    res = talk_to_penpal(agent, user_input, params)
-    return res
-
-
-st.title("EF PenPal 🤖 ")
```
Comment (hansbantilan) [inline generative_agents/demo/penpal_app.py:19]: With this file renaming, may I please request an update of our README to have
`streamlit run penpal_app/app.py`

File: generative_agents/modeling/agent_helpers.py [hunk 2]
```diff
@@ -24,20 +24,26 @@ def load_pinecone():
         pinecone.create_index("generative-agents-index", dimension=1536)
 
 def load_agent(params, llm):
-    log.info(f"Instantiating agent: {params['name']}...")
+    name = params["name"]
+    log.info(f"Instantiating agent: {name}...")
+    backstory = params["backstory"].replace("EF-PenPal", name)
```
Comment (hansbantilan) [inline generative_agents/modeling/agent_helpers.py:29]: interesting, nice way of customizing the name!

File: generative_agents/utility/delete_index.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import pinecone
+import os
+import argparse
+
+from generative_agents.utility import logger
+
+log = logger.init("delete_index")
+
+def delete_index(index_name):
```
Comment (hansbantilan) [inline generative_agents/utility/delete_index.py:9]: Nice! Thanks for adding the delete_index() functionality here

File: generative_agents/utility/utility.py [hunk 4]
```diff
@@ -1,7 +1,16 @@
...
     with open(path) as f:
         params = yaml.safe_load(f)
     return params
+
+def get_completion(prompt, model="gpt-3.5-turbo"):
+    messages = [{"role": "user", "content": prompt}]
+    response = openai.ChatCompletion.create(
+        model=model,
+        messages=messages,
+        temperature=0, # this is the degree of randomness of the model's output
```
Comment (hansbantilan) [inline generative_agents/utility/utility.py:14]: Good choice of temperature=0 for now, for reproducibility

**Review by hansbantilan — APPROVED** [review]
app.py is so clean, thank you Amrita. 

Let's get this in!

File: generative_agents/demo/penpal_app.py (hunk 1 above)
Comment (amritap-ef) [reply generative_agents/demo/penpal_app.py:19]: Done!

**Pull request merged**